NOTES.md

```diff
@@ -1,3 +1,246 @@
+# Release 1.16.0 (Stand By Me)
+
+Released on 2015-01-29
+
+The highlights of this release are:
+* A new administration API
+* Changefeeds on complex queries
+* Numerous improvements and enhancements all over the place
+
+Read the [release blog post][1.16-blog] for more details.
+
+[1.16-blog]: http://rethinkdb.com/blog/1.16-release/
+
+## Compatbility ##
+
+### Backwards-compatible changes ###
+
+Data files from RethinkDB versions 1.13.0 onward will be automatically
+migrated to version 1.16.x. As with any major release, back up your data files
+before performing the upgrade. If you are upgrading from a release earlier
+than 1.13.0, follow the migration instructions before upgrading:
+
+http://rethinkdb.com/docs/migration/
+
+Secondary indexes now use a new format; old indexes will continue to work, but
+you should rebuild indexes after upgrading to 1.16.x. A warning about outdated
+indexes will be issued on startup.
+
+Indexes can be migrated to the new format with the `rethinkdb index-rebuild`
+utility. Consult the troubleshooting document for more information:
+
+http://rethinkdb.com/docs/troubleshooting#my-secondary-index-is-outdated
+
+The abstraction of datacenters has been replaced by [server tags][server-tags-documentation].
+Existing datacenter assignments will be converted to server tags automatically.
+
+[server-tags-documentation]: http://rethinkdb.com/docs/sharding-and-replication/
+
+### API-breaking changes ###
+
+The `tableCreate`, `tableDrop`, `dbCreate` and `dbDrop` terms have a new set of return values.
+The previous return values `created` and `dropped` have been renamed to `tables_creates` / `dbs_created` and
+`tables_dropped` / `dbs_dropped` respectively. The terms now additionally return a `config_changes` field.
+
+Consult the API documentation for these commands for details:
+[tableCreate][tableCreate-api-docs], [tableDrop][tableDrop-api-docs], [dbCreate][dbCreate-api-docs], [dbDrop][dbDrop-api-docs]
+
+[tableCreate-api-docs]: http://rethinkdb.com/api/javascript/tableCreate
+[tableDrop-api-docs]: http://rethinkdb.com/api/javascript/tableDrop
+[dbCreate-api-docs]: http://rethinkdb.com/api/javascript/dbCreate
+[dbDrop-api-docs]: http://rethinkdb.com/api/javascript/dbDrop
+
+Changefeeds on a table now combine multiple changes to the same document into a single notification if they happen rapidly.
+You can turn this behavior off by passing the `squash: false` optional argument.
```

Comment:
I would collapse this sentence (it's confusing since you see "Consult the API documentation" on its own twice in this section):

> You can turn this behavior off by passing the `squash: false` optional argument (see the [API documentation][changes-api-docs] for more details).